### PR TITLE
feat(ColorToolkit): Update naming for tag constants

### PR DESCRIPTION
**Pull request by idenjoe**
🎟️ Jira ticket

[IOS-7356](https://jira.tid.es/browse/IOS-7356) Global naming for tag constants

🥅 What's the goal?

- Modify some tag constant names: https://github.com/Telefonica/mistica-design/pull/456
- Remove textAmount tag
- New constants for MOVES: https://github.com/Telefonica/mistica-design/pull/475/files
- Reviewed all the palettes with their corresponding colors and also reviewed each of the colors of each palette

🚧 How do we do it?

Renamed all constants with their new names. Added the new colours for MOVES.

🧪 How can I verify this?

### CODE REVIEW

**Comment by yceballost** [comment]
can you export a Mistica Catalog release to review?

**Comment by idenjoe** [comment]
> can you export a Mistica Catalog release to review?

@yceballost there is a problem building the Catalog. @DavidMarinCalleja will check it next week.

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ github "Telefonica/mistica-ios"
 And then run:
 
 ```
-carthage update --platform iOS --use-ssh
+carthage update --platform iOS --use-ssh --use-xcframeworks
```
Comment (jmbrocal) [inline README.md:52]: Why do we need this change?

**Review by yceballost — APPROVED** [review]
everything seems correct! thanks guys!

**Review by DavidMarinCalleja — APPROVED** [review]

**Pull request merged**